website/contributing/contributing-intro.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Contributing Intro
```

Comment:
It was the earlier name of the page, but I feel like it doesn't suit the content that good, also it looks odd with other How To's in the sidebar. 

The idea behind the name was to reflect the summary/overview content in there. What do you think about, for example: "Contributing Overview"? But I'm not sure if this is more formal though, however feel free to post you ideas, I'm not that good at naming. 😅

Reply:
Maybe we can go with "Overview", since we already have one for the Architecture: 
* https://reactnative.dev/architecture/overview

However, I also like the "Contributing Introduction" suggestion, it sounds a bit more formal to me. 

Let's wait what @cortinico prefer, then I update the title, filename, links and redirect.

Reply:
I'm fine either way. Just "Contributing" as a landing page title also works as it's a known keyword in the OSS space (i.e. you have a CONTRIBUTING.md file).

Otherwise "Contributing Overview" works fine for me 👍